test/python/importer/jit_ir/ivalue_import/tensors_non_value_semantics.py

```diff
@@ -5,7 +5,7 @@
 import typing
```

Comment:
I think you can just leave this one as tensors.py

Reply:
changed.